docs/api-docs/catalog/v2-vs-v3.md

```diff
@@ -0,0 +1,131 @@
+# Difference between V2 and V3 Catalog REST APIs
+
+<div class="otp" id="no-index">
+
+### On this page
+- [V3 improvements](#v3-improvements)
+- [Difference between V2 and V3 Catalog APIs](#difference-between-v2-and-v3-catalog-apis)
+- [Related resources](#related-resources)
+
+</div>
+
+V2 and V3 Catalog REST APIs allow you to manage your store's products, categories, and brands, along with their sub-resources.
+
+Both the V2 and V3 Catalog APIs authenticate with OAuth and can be used concurrently within a single application; however, data representation can be significantly different.
+
+V2 and V3 REST APIs are not fully compatible. When resources are available through both APIs, we recommend using the V3 API as it contains performance optimizations and usability improvements.
+For differences in resources, see the [Difference between V2 and V3 Catalog APIs](#difference-between-v2-and-v3-catalog-apis) section of this article.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
+>
+> * BigCommerce will not introduce new resources in V2 APIs.
+> * BigCommerce will eventually migrate all existing V2 resources to V3.
+> * Not all V2 resources have V3 counterparts. 
```

Comment:
I think we should also point out that all V2 Catalog REST APIs are deprecated (link to deprecations and sunsets article) and all existing users of V2 are encouraged to move to V3.